### ISSUE TITLE
Scrolling issues when using overflow-scroll="true"

### ISSUE THREAD

**Issue by nokturnal** [driftyco/ionic#2411]
A full thread on the Ionic forum as discussed this issue: http://forum.ionicframework.com/t/bug-overflow-scroll-not-working-in-beta-12-beta13/9514/18

Test projects and suggestions have also been suggested in that thread as well.

As a summary, overflow-scroll="true" on ion-content has stopped working in newer versions of the framework.  The native scrolling will not work with a single finger, but works with two fingers.

@mhartington : Can you chime in with your experience with this issue?

Cheers

**Comment by rahul54784** [comment]
after add this line in config.xml solve problem in ios 
 < preference name="DisallowOverscroll" value="false" />

**Comment by Oregand** [comment]
rahul54784, You're the man.

**Comment by dioxmio** [comment]
is that solved??? I have tried with 1.2.4 and I can see that when I use `overflow-scroll="true"` some nodes get lost whereas if I remove it no nodes are lost (I'm just switching over two screen, super simple scenario). I have tried `v1.2.4-nightly-2998` and whereas it seems to be a bit better the leak is still there. I would like to reduce as much as possible since we developing for mobile.

many thanks,

jose
